src/lightning_lite/strategies/fsdp.py

```diff
@@ -258,7 +257,6 @@ def broadcast(self, obj: TBroadcast, src: int = 0) -> TBroadcast:
     def register_strategies(cls, strategy_registry: Dict) -> None:
         if not _TORCH_GREATER_EQUAL_1_12 or not torch.distributed.is_available():
             return
-        from torch.distributed.fsdp.fully_sharded_data_parallel import CPUOffload
```

Comment:
After this change, when would the user want to instantiate and pass this class? What other arguments does it expose?

Reply:
> After this change, when would the user want to instantiate and pass this class?

Probably never, but it was the only way so far. It depends on the PyTorch team to decide whether they see other useful arguments to be included in the dataclass or not. My guess is not, they will probably eventually go with a bool (after all, the activation checkpointing has moved to a separate API as well).